.eslintrc.js

```diff
@@ -73,7 +73,6 @@ const bestPracticesRules = {
 	'accessor-pairs': ERROR,
 	'array-callback-return': ERROR,
 	'block-scoped-var': ERROR,
-	'class-methods-use-this': TRANSITION_WARNING,
```

Comment:
There are a bunch of places we don't do this and I'm not sure we need to enforce it.